pdf/model/image.go

```diff
@@ -251,32 +250,42 @@ func (ih DefaultImageHandler) NewImageFromGoImage(goimg goimage.Image) (*Image,
 	default:
 		// Speed up jpeg encoding by converting to RGBA first.
 		// Will not be required once the golang image/jpeg package is optimized.
-		m = goimage.NewRGBA(goimage.Rect(0, 0, b.Dx(), b.Dy()))
-		draw.Draw(m, m.Bounds(), goimg, b.Min, draw.Src)
+		b = goimage.Rect(0, 0, b.Dx(), b.Dy())
+		m = goimage.NewRGBA(b)
+		draw.Draw(m, b, goimg, b.Min, draw.Src)
```

Comment:
There is a potential bug here by rewriting b before copying because if b.Min was anything other than 0, 0 before, the copy would be incorrect (not sure if this ever happens in practice but it's technically possible). That's the reason why I suggested:

```
m = goimage.NewRGBA(goimage.Rect(0, 0, b.Dx(), b.Dy()))
draw.Draw(m, m.Bounds(), goimg, b.Min, draw.Src)
b = m.Bounds()
```

Reply:
Nice catch.  Fixed.